### PR TITLE
blinQ: Template updates, support for Enterprise Scale module v3.0.0

### PR DESCRIPTION
**Pull request by heintonny**
# [Issue-id](https://github.com/Azure/caf-terraform-landingzones/issues/425)

## PR Checklist

---

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [X ] My code follows the code style of this project.
- [X ] I ran lint checks locally prior to submission.
- [] Have you checked to ensure there aren't other open Pull Requests for the same update/change?

## Description

This pull request might overlap with some of the commits in PR 436. We probably need to discuss this PR in our next meeting.
Main topic for this PR is:
- Fix some bugs in Ansible template for multi subscription deployment
- Add support for enterprise scale module v3.0.0 to deploy core landing zones and policy
- Template workaround for: https://github.com/microsoftgraph/msgraph-metadata/issues/92
- Add support for sub_sandbox in template
- Add Ansible logic to create private endpoints and user defined routes. This is required so that lz resources don't brake policy requirements and downgrade the security score. Target should be 100% score for lz resources, this needs some more work and discussions about the bootstrap process and when and how public endpoints should be closed by Ansible configuration updates 
- +++

MS might not want to include all these commits based on target architecture decisions, but we belive most of the contribution will benefit the community. 

## Does this introduce a breaking change

- [X] YES
- [ ] NO

## Testing
We have tested the template version locally

### CODE REVIEW

**Comment by heintonny** [comment]
Changed base to int-5.6.0 branch that we used as a starting point.

File: templates/ansible/walk-through-bootstrap.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     - name: customer_name
       prompt: Set the short version of your customer name with no spaces
       private: no
-      default: contoso
```
Comment (heintonny) [inline templates/ansible/walk-through-bootstrap.yaml:19]: No point to change the default, let's keep the contoso

File: templates/ansible/walk-through-bootstrap.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@
     - name: prefix
       prompt: Set the prefix to add to all resource.
       private: no
-      default: caf
```
Comment (heintonny) [inline templates/ansible/walk-through-bootstrap.yaml:29]: No point to change the default, let's keep caf

**Comment by LaurentLesle** [comment]
Will not merge it as 5.6.9 will include it

**Pull request closed without merging**